Sender/data_from_csv.cpp

```diff
@@ -0,0 +1,53 @@
+#include"data_from_csv.h"
+
+DataFrom_csv::DataFrom_csv(std::string& fileName): _csvReader(fileName)
+{	if (_csvReader.ReadIsGood)
+	{	_file_lines = _csvReader.GetFileLines();
+		FormatLines_csv(); 
+	}
+}
+void DataFrom_csv::FormatLines_csv()	
+//needs to be changed if input file format is different
+{	UpdateOutputFormat();
+	UpdateMonthAndYear();
+	UpdateFootfallValues();
+}
+void DataFrom_csv::UpdateOutputFormat()
+{	const char* outputFormat = _file_lines[0].c_str();
+	_output_format.UpdateFormat(outputFormat);
+}
+void DataFrom_csv::UpdateMonthAndYear()
+{	std::vector<int> month_and_year = SplitString(_file_lines[1]);
```

Comment:
always good to add `defensive checks` to make sure the `vector is not empty` before accessing the index.